### ISSUE TITLE
Unexpected setting owner of log_dir/* if logfile is given

### ISSUE THREAD

**Issue by blelump** [DavidWittman/ansible-redis#83]
Hi, 

I've specified log file as `redis_logfile: /var/log/redis.log` and it caused all the remaining log files in `/var/log` have changed owner to redis, e.g:

```
-rw-r-----  1 redis    adm             0 lis 27 06:38 syslog
```

which essentially caused other non–root services either to crash or unable to log .

**Comment by DavidWittman** [comment]
@blelump I'm sorry this impacted you in such a way. I'm in the process of testing out some fixes for this issue.

**Comment by DavidWittman** [comment]
Closed by #87

**Comment by blelump** [comment]
@DavidWittman, thanks!
